packages/expo-camera/src/Camera.tsx

```diff
@@ -117,10 +118,16 @@ export default class Camera extends React.Component<CameraProps> {
     whiteBalance: CameraManager.WhiteBalance.auto,
   };
 
+  /**
+   * @deprecated use `getCameraPermissionsAync` instead.
```

Comment:
I'm not sure if this is right, I think it is (since the native `getPermissionsAsync` is exactly the same as `getCameraPermissionsAsync`). I can revert this, if you want!

Reply:
@lukmccall I added the warning, based on the `expo-location` warnings. Also added another changelog entry as breaking change for better exposure 😄  Is this good to go?

Reply:
Yes 😉